frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -33,22 +35,30 @@ const ArticleForm: React.FC = () => {
   const handleArticleSubmit = useCallback(
     (articleSubmitType: ValueOf<typeof ArticleSubmitType>) =>
       (payload: ArticleRequestDto): void => {
+        const isArticlePublish =
+          articleSubmitType === ArticleSubmitType.PUBLISH;
+
         const updatedPayload = {
           ...payload,
-          publishedAt:
-            articleSubmitType === ArticleSubmitType.PUBLISH
-              ? new Date().toISOString()
-              : null,
+          publishedAt: isArticlePublish ? new Date().toISOString() : null,
         };
 
         void dispatch(
           articlesActions.createArticle({
             articlePayload: updatedPayload,
             generatedPrompt: getGeneratedPromptPayload(generatedPrompt),
           }),
-        );
+        )
+          .unwrap()
+          .then(() => {
+            navigate(
+              isArticlePublish
+                ? AppRoute.ARTICLES
+                : `${AppRoute.ARTICLES}/${ArticleSubRoute.MY_ARTICLES}`,
```

Comment:
I guess this slash should be in MY_ARTICLES enum value

Reply:
In such case I get an error, as I understood it is because the slash is inserted automatically
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/84803938/303490a9-54e1-40f0-ba7c-cf9335c1681c)

Reply:
Got it 👍